ansible/roles/es-azure-snapshot/tasks/main.yml

```diff
@@ -17,8 +17,7 @@
       {"indices":"*","include_global_state":false}
     headers:
       Content-Type: "application/json"
-  async: 45000
-  poll: 10
+    timeout: 60
```

Comment:
@santosh0705 This will time out the operation and will return an error. Why do we have to do this ?

Reply:
Async will only help in long running ssh connection. We are not doing any such ssh operation and it will never help here. This code was introduced in commit d5169f2 and even after this change we are having request timeout, check the job logs. I removed this since I find this is absolute unnecessary. Let me know if you are disagree with this.
BTW I've only suggested to increase the timeout for URI module for this particular request. Check commit f3078c9

Reply:
1. Async/poll will check the job status at particular time interval(poll) with a timeout(async). Using that we'll get to know, what's the status of the job and then wait/exit with an error.
2. But timeout will blatantly wait(even if the tas completed in 10s, in this case, it'll take minutes)
@santosh0705

Reply:
`timeout` is a parameter to URI module and the URI module (the GET request) is timing out not the Ansible job. Doc here: https://docs.ansible.com/ansible/latest/modules/uri_module.html
We are doubling the default timeout so the URI module will wait for the HTTP response and not just terminate.
You can have the async/poll on but timeout is required for URI module.

Reply:
@santosh0705 I am closing this pr, as timeout won't fix this issue.

Reply:
@rjshrjndrn Thank you